build.gradle

```diff
@@ -1,7 +1,7 @@
 buildscript {
     ext.ktlintVersion = '0.48.0'
-    ext.kotlinVersion = '1.9.20'
-    ext.androidApplicationVersion = '8.6.1'
+    ext.kotlinVersion = '1.9.21'
```

Comment:
can just go up to 1.9.24 and compose compiler version 1.5.14 https://developer.android.com/jetpack/androidx/releases/compose-kotlin

Reply:
OK.. updated... 

1.9.21 is what Android Studio prompts for.. but we can move faster.